src/metrics/pocket_area/base.py

```diff
@@ -11,6 +11,7 @@ class PocketAreaMetadata:
     vertices: Optional[List[Point]] = None
     radius: Optional[float] = None
     center: Optional[Point] = None
+    edge: Optional[Point] = None
```

Comment:
Unit tests are failing on this PR because `edge` is a new property and there are tests that check converting a pocket area to JSON that are now not correct because the actual data returns `edge` but their expected data does not have it.

However, I don't think we need edge, so it will probably be easiest to just remove edge from the metadata class and code, and then the tests will be passing.